src/components/TipInput.vue

```diff
@@ -248,6 +248,9 @@ export default {
       }
       return i18n.t('components.TipInput.totalRetips');
     },
+    canTip() {
+      return this.isLoggedIn && !this.loading.wallet;
```

Comment:
IMHO we need to have a separate case where wallet is loading in this case it should not use deeplink but rather disable the button

Reply:
There is no indication that a user is being authenticated. The process in not straightforward. I think it would be more confusing if buttons are enabled and disabled on the background. In this implementation are always doing their purpose - tip/retip.